### DISCUSSION TITLE
通过misx文件安装出现证书错误

### DISCUSSION THREAD

**Discussion by niconihe** [xiaoyaocz/flutter_dmzj#163] [General]
<img width="490" alt="image" src="https://user-images.githubusercontent.com/29039907/231195487-9d461693-1990-44d9-9b59-8375a57aaaa8.png">

**Comment by xiaoyaocz** [comment]
右键文件-属性-数字签名-选中-详细信息-查看证书-安装证书-本地计算机-将证书放到"受信任的根证书颁发机构"-下一步，完成
